pkg/stable.packages.props

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+
+    <!-- NETCore 1.1.0 core-setup -->
+    <StablePackage Include="Microsoft.NETCore.App">
+      <Version>1.1.0</Version>
+    </StablePackage>
+    <StablePackage Include="Microsoft.NETCore">
+      <Version>5.0.3</Version>
+    </StablePackage>
+    <StablePackage Include="Microsoft.NETCore.UniversalWindowsPlatform">
```

Comment:
The only meta-package that we want to mark stable is Microsoft.NETCore.App. We don't want to marke the others as stable.